code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -226,7 +225,6 @@
 	weight = 3
 	cost = 35
 	requirements = list(101,101,80,70,60,60,50,50,40,40)
-	high_population_requirement = 35
```

Comment:
this is the only use of this var that seems intentional and sane; i'd change the final entry in the requirements list to 35

Reply:
This is a thing that can be changed in the configs so Ill leave it be.